webapp/client/src/lib/translations.js

```diff
@@ -95,6 +95,14 @@ const translations = {
         "Mit Ihrer Registrierung beantragen Sie einen Freischaltcode bei Ihrer Finanzverwaltung. Sie erhalten diesen mit einem Brief <bold>innerhalb von zwei Wochen</bold> nach erfolgreicher Beantragung. Wenn Sie die Zusammenveranlagung nutzen möchten, muss sich nur eine Person registrieren.",
     },
   },
+  unlockCodeRevocation: {
+    idnr: {
+      labelText: "Steuer-Identifikationsnummer",
+    },
+    dob: {
+      labelText: "Geburtsdatum",
+    },
+  },
```

Comment:
I am just wondering if we really should split the translations in that way. Wouldn't it make more sense to set the label for the IDNr only once and reuse it for all the different flows. If it changes for the unlockCodeInput, it should also change here, shouldn't it?

Reply:
Mh..maybe. In that case it should be directly set on the IdNr field then though to be consistent, I think? And then, when we do that, it's also only consistent to do that with all fields. And I'm not sure we want a PrenameField and a SurnameField etc.? But maybe thats. too dogmatic of me^^

Reply:
Maybe, I did get your answer wrong but I think that we have a misunderstanding. I think there are two ways to think about it:
1.  Set it once in a specific component
2. Set it only once in the translations and do not connect it with a specific flow

I meant the second option. That would basically mean to change translations like this (or without the components) and then use that identifier anywhere we use a idnr field:
```
 components: {
    idnr: {
      labelText: "Steuer-Identifikationsnummer",
    },
    dob: {
      labelText: "Geburtsdatum",
    },
  },
```

Reply:
I called moved it into `fields` now as that already existed. I suppose that's fine for you?